scripts/check-all.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo "Running check-all"
+
+pnpm lint-branch &
+lint_status=$?
+
```

Comment:
I don't think this will work.  

```
pnpm lint-branch &
```

causes it to go to the background but there's no exit code yet.

Then you store it here:

```
lint_status=$?
```

But the code is probably zero.  There's a race condition.

The only way you could do this is to NOT run ```set -e``` 

... then run each command until completion (not in the background), then store the output.

Reply:
Good catch, I will handle the race condition more explicitly